src/Tizen.NUI.Components/Style/ButtonStyle.cs

```diff
@@ -89,6 +82,7 @@ public class ButtonStyle : ControlStyle
         public static readonly BindableProperty IconPaddingProperty = BindableProperty.Create(nameof(IconPadding), typeof(Extents), typeof(ButtonStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
         {
             var buttonStyle = (ButtonStyle)bindable;
+            if (null == buttonStyle.iconPadding) buttonStyle.iconPadding = new Extents(buttonStyle.OnIconPaddingChanged, 0, 0, 0, 0);
```

Comment:
Could you help me to understand this code?  
After setting `buttonStyle.iconPadding` to new Extents, the next line is re-setting it to `(Extents)newValue`.

Reply:
Sorry, should use copyfrom.